### PR TITLE
Add the ability to destroy a cluster

### PR DESCRIPTION
**Pull request by hamzy**
This is written entirely in Go.  The shell script was used as
an example.

### CODE REVIEW

File: destroy-cluster/go.mod [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+go 1.17
+
+require (
+	github.com/IBM-Cloud/bluemix-go v0.0.0-20211006063826-ee9901463fa3
+	github.com/IBM/go-sdk-core/v4 v4.10.0
+	github.com/IBM/go-sdk-core/v5 v5.7.0
+	github.com/IBM/networking-go-sdk v0.24.0
+	github.com/IBM/platform-services-go-sdk v0.21.0
+	github.com/IBM/vpc-go-sdk v1.0.1
+	github.com/ppc64le-cloud/pvsadm v0.1.3
```
Comment (mkumatag) [inline destroy-cluster/go.mod]: As I mentioned earlier, `github.com/ppc64le-cloud/pvsadm` repo is not meant for exporting outside the project, let's not depend on this repo here, there is no guarantee we can give for the clients(they may change frequently)

**Review by mkumatag — CHANGES_REQUESTED** [review]
whats the reason behind using this instead of the scripts written? are these just experimental scripts we are landing here before we merge in the installer code?

File: destroy-cluster/go.mod (hunk 1 above)
Comment (hamzy) [reply destroy-cluster/go.mod]: Removed

**Comment by hamzy** [comment]
Yes, these are experimental "scripts" landing here to document history and so they don't get lost before merging in the installer code.

**Comment by hamzy** [comment]
It shouldn't be this hard to add code to the project.
I created my own repo to save it https://github.com/hamzy/powervs-hack

**Pull request closed without merging**